doc: Update versions for 2.2.0 release (#877)
nrfconnect/sdk-nrfxlib: nrf_dm/doc/CHANGELOG.rst

> All the notable changes to this project are documented in this file.

Reviewer: ```suggestion
All the notable changes to this project are documented on this page.
```
